controllers/idler/idler_controller_test.go

```diff
@@ -240,13 +245,14 @@ func TestEnsureIdling(t *testing.T) {
 					StatefulSetScaledDown(podsRunningForTooLong.statefulSet).
 					StatefulSetScaledUp(podsTooEarlyToKill.statefulSet).
 					StatefulSetScaledUp(noise.statefulSet).
+					StatefulSetScaledUp(podsCrashLoopingWithinThreshold.statefulSet).
 					VMStopped(podsRunningForTooLong.virtualmachine).
 					VMRunning(podsTooEarlyToKill.virtualmachine).
 					VMRunning(noise.virtualmachine)
 
-				// Still tracking all pods. Even deleted ones.
+				// Only tracks pods that have not been processed in this reconcile
```

Comment:
Not been processed or rater not deleted yet?

Reply:
updated in [82f8118](https://github.com/codeready-toolchain/member-operator/pull/630/commits/82f8118a99f81e4c34d39486a26b46c419193af3),  have also added a comment in the third reconcile loop where we track controlledpods again